vivisect/tests/testvivisect.py

```diff
@@ -1057,8 +1066,8 @@ def test_chgrp_exports(self):
         vw = self.chgrp_vw
         exports = [
             # .rodata
-            (0x08050650, 4, v_const.LOC_POINTER, 'chgrp._fp_hw'),
-            (0x08050654, 4, v_const.LOC_POINTER, 'chgrp._IO_stdin_used'),
+            (0x08050650, 4, v_const.LOC_NUMBER, 'chgrp._fp_hw'),  # 08050650  uint32_t _fp_hw = 0x3
+            (0x08050654, 4, v_const.LOC_NUMBER, 'chgrp._IO_stdin_used'),  # 08050654  uint32_t _IO_stdin_used = 0x20001
```

Comment:
these entries are now correctly typed (they are not pointers, despite what the test was asserting).